pywbemtools/pywbemcli/_cmd_statistics.py

```diff
@@ -0,0 +1,443 @@
+
+
+@statistics_group.command('server-off', cls=PywbemcliCommand,
+                          options_metavar=CMD_OPTS_TXT)
+@add_options(help_option)
+@click.pass_obj
+def statistics_server_off(context):
+    """
+    Disable statistics on current server.
+
+    This command turns off the gathering of server statistics in the WBEM
+    server and the return of server response times for inclusion in
+    pywbemcli maintained statistics. The gathering of server statistics and the
+    returning of server response times may not be implemented by all WBEM
+    servers.
+
+    This may fail if the server does not manage statistics or does not allow
+    a client to modify the state of statistics gathering.
+
+    WBEM server statistics gathering state does not impact pywbemcli client
+    statistics gathering other than whether the 'Server Times` row is included
+    in the report. See the --timestats general option.
+    """
+    # pylint: disable=too-many-function-args
+
+    context.execute_cmd(lambda: cmd_statistics_server_off(context))
+
+
+@statistics_group.command('server-show', cls=PywbemcliCommand,
+                          options_metavar=CMD_OPTS_TXT)
+@add_options(help_option)
+@click.pass_obj
+def statistics_server_show(context):
+    """
+    Display statistics gathered by server.
+
+    Display the current statistics gathered in the WBEM server if statistics
+    gathering is implemented and active (see ```statistics server-on```) for
+    the current WBEM server.
+
+    These statistics are independent of the statistics gathered by the pywbemcli
+    client and displayed for example with the command ``statistics show``.
+
+    Presents a table of the gathered statistical information.
+
+    This command is not implemented. See issue #895
+    """
+    # pylint: disable=too-many-function-args
+    context.execute_cmd(lambda: cmd_statistics_server_show(context))
+
+
+@statistics_group.command('show', cls=PywbemcliCommand,
+                          options_metavar=CMD_OPTS_TXT)
+@add_options(help_option)
+@click.pass_obj
+def statistics_show(context):
+    """
+    Display statistics managed by pywbemcli.
+
+    Display the current statistics including client managed statistics on WBEM
+    operations and if WBEM server statistics are implemented and enabled, the
+    statistics on the returned ``WBEMServerResponseTime`` header.
+
+    Statististics are always displayed as a table independent of output-format
+    defined.
+    """
+    # pylint: disable=too-many-function-args
+    context.execute_cmd(lambda: cmd_statistics_show(context))
+
+
+@statistics_group.command('status', cls=PywbemcliCommand,
+                          options_metavar=CMD_OPTS_TXT)
+@add_options(help_option)
+@click.pass_obj
+def statistics_status(context):
+    """
+    Show statistics enabled status for client and server.
+
+    Show enabled status on statistics gathering for this client and the
```

Comment:
On "this client": Usually we say just "pywbemcli".

Reply:
DONE